cirq/ops/phased_x_gate_test.py

```diff
@@ -28,7 +28,14 @@ def test_phased_x_consistent_protocols(phase_exponent):
     cirq.testing.assert_implements_consistent_protocols(
             cirq.PhasedXPowGate(phase_exponent=phase_exponent,
                                 exponent=1.0,
-                                global_shift=0.1)
+                                global_shift=0),
+            is_global_phase_sensitive = False
+    )
+    cirq.testing.assert_implements_consistent_protocols(
```

Comment:
Why the duplicated test? The global phase sensitive version passing implies the global phase insensitive version will pass. Undo changes to this file.

Reply:
It's not duplicated. The phase insensitive version passes only if global_shift==0. 
Added the insensitive test to verify that global phase doesn't change if the arg is 0.